extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -917,6 +917,24 @@ export const popupMessageListener = (request: Request) => {
     return { error: "Session timed out" };
   };
 
+  const signFreighterSorobanTransaction = () => {
+    const { transaction, network } = request;
+
+    const transaction2 = SorobanSdk.TransactionBuilder.fromXDR(
```

Comment:
Ah, I came up with these var names quickly while debugging and I did kind of a bad job here tbh. Would you mind following the pattern from ln 904 above?

It would be more descriptive and easier to follow if it were:

```
    const { transactionXDR, network } = request;
    
    const transaction = SorobanSdk.TransactionBuilder.fromXDR(
```

Thanks and sorry!! You'll need to, unfortunately, rename the params in all the function invocations